package.json

```diff
@@ -73,7 +73,7 @@
     "hot-shots": "^10.0.0",
     "http-errors": "~1.8.0",
     "is-svg": "^4.4.0",
-    "isomorphic-dompurify": "^0.27.0",
+    "isomorphic-dompurify": "^2.0.0",
```

Comment:
seems to be 2 major versions ahead, any breaking changes?

Reply:
Idt it affects us tbh